bsi/src/main/java/org/roaringbitmap/bsi/BitmapSliceIndex.java

```diff
@@ -41,45 +35,35 @@ enum Operation {
 
   long getLongCardinality();
 
-
   /**
-   * set value for bsi, setValue will set each bit slice according to the input value
-   * given that we have bsi as follow
-   * ebm:  RB[1 1 1 1]
-   * slice0: RB[0 1 0 1]   -
-   * slice1: RB[1 0 1 0]  |
-   * slice2: RB[1 0 0 1]  |--bA:bit slice Array
-   * slice3: RB[0 0 0 1]   -
-   *      1 2 3 4
-   *      |
-   *      ---------------- columnId or rowId
-   *  for columnId 1, the value is 110 that is :6
-   *  for columnId 2, the value is 1 that is :1
-   *  for columnId 3, the value is 10 that is :2
-   *  for columnId 4, the value is 1101 that is :11
+   * set value for bsi, setValue will set each bit slice according to the input value given that we
```

Comment:
@blacelle That's kind of messed up... the comment is no longer understandable because of the reformat.

Is there a way to tell it not to reformat the comments beyond, maybe, breaking long lines?

Reply:
you can do `googleJavaFormat().formatJavadoc(false)`

the underlying issue is that javadoc is supposed to be html-ish, I think the rendered HTML Javadoc here will also look weird, and it would ideally be `<ol><li>` etc. If you put in the right HTML tags, then I think gjf will format it nicely too. In this case I think the formatting is sort of "catching a bug", but perfectly reasonable to ignore it too.